Src/WindowsServer/WindowsServer.Shared.Tests/FirstChanceExceptionStatisticsTelemetryModuleTest.cs

```diff
@@ -0,0 +1,204 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.ExceptionServices;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.Web.TestFramework;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using TestFramework;
+    using Assert = Xunit.Assert;
+    
+    [TestClass]
```

Comment:
nitpick: Whoever is writing code with no comments should be required to comment it.  This is public code and making it readable for others to consume matters.